### DISCUSSION TITLE
full Screen area of Android Air

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#1638] [General]
is it possible to fully use the screen area in air android. At the moment there is a full-screen mode, but there are black unused areas on the left and on the right. How to get rid of them, (use the entire screen area completely)
![Безымянный-1](https://user-images.githubusercontent.com/98939255/153540579-287bdc09-278c-4328-b7fa-d8d949867431.png)

**Comment by marchbold** [comment]
Try adding 

```
<meta-data android:name="android.max_aspect" android:value="2.5" />
```

inside your application tag in your manifest additions

```xml
<android>
    <manifestAdditions><![CDATA[
        <manifest android:installLocation="auto">

            
            <application>
                <meta-data android:name="android.max_aspect" android:value="2.5" />
            </application>
        
        </manifest>
    ]]></manifestAdditions>
</android>
```


https://docs.airnativeextensions.com/docs/application/display/#aspect-ratios
